Tests/eppo/ClientTests.swift

```diff
@@ -7,10 +7,10 @@ import OHHTTPStubsSwift
 @testable import eppo_flagging
 
 let fileURL = Bundle.module.url(
-    forResource: "Resources/test-data/rac-experiments-v3.json",
+    forResource: "Resources/test-data/ufc/flags-v1.json",
```

Comment:
📈

Reply:
Note this is the unobfuscated version; I imagine a subsequent PR will deal with all the obfuscation fun 😅